boards/st/stm32h7b3i_dk/Kconfig.defconfig

```diff
@@ -22,10 +22,10 @@ if LVGL
 config CACHE_MANAGEMENT
 	default y
 
-config LV_USE_GPU_STM32_DMA2D
+config LV_USE_DRAW_DMA2D
 	default y
 
-config LV_GPU_DMA2D_CMSIS_INCLUDE
+config LV_DRAW_DMA2D_HAL_INCLUDE
 	default "stm32h7xx.h"
```

Comment:
Note: We'll need to move this to `soc/st/stm32/stm32h7x/Kconfig.defconfig` as it's common to whole STM32H7 series so it doesn't make sense to put in board config.

Reply:
Make this part of this PR or a susequent one?

Reply:
No need to address that now as this is not related with this change. It can wait.

Reply:
DMA2D support in LVGL 9.2 might need extra code to work.

Reply:
If I understand [dma2d-support](https://docs.lvgl.io/master/details/integration/chip/stm32.html#dma2d-support) correctly to benefit from dma2d we need to use `LV_USE_DRAW_DMA2D_INTERRUPT ` with `LV_USE_OS` and call `lv_draw_dma2d_transfer_complete_interrupt_handler` when "DMA2D "transfer complete" global interrupt is received". Therefore, extra interrupt setup and handling code is required.

Reply:
Enabling `LV_Z_USE_OSAL` is already possible and together with dma2d should already give some performance improvement. Its true that the interrupt setup is needed to avoid actively polling for completion.

Since I do not own a device with chromart I can not test this easily. If someone wants to help out on this I appreciate every bit :^)